components/rowListScene.brs

```diff
@@ -1,12 +1,21 @@
 sub init()
-    m.global.addField("http", "node", FALSE)
     m.global.addField("ratio", "float", FALSE)
-    m.global.http = createObject("roSGNode", "httpTask")
-    m.global.http.control = "RUN"
+    m.flag = true
     m.global.ratio = 1
     m.rowList = m.top.findNode("mainRowList")
+    m.MainBoldLabel = m.top.findNode("MainBoldLabel")
+    m.DateAndDurationLabel = m.top.findNode("DateAndDurationLabel")
+    m.DescriptionLabel = m.top.findNode("DescriptionLabel")
+    m.CategoryLabel = m.top.findNode("CategoryLabel")
+    m.fadeInAnimation = m.top.findNode("fadeInAnimation")
+    m.heroDetailsId = m.top.findNode("heroDetailsId")
```

Comment:
We will not need this line if we use the above optimization.

Reply:
Alright